### ISSUE TITLE
Different Result, Same Settings and Seed

### ISSUE THREAD

**Issue by ghost** [FuouM/HidamariDiffusionColab#1]
Hello, I ported your notebook to work locally [using the same fork of stable diffusion](https://github.com/FuouM/stable-diffusion-hidamari), I didn't change any of the generation scripts and I am using the exact same 1,4 model from huggingface but for some reason I am getting a different results for the same configuration.

I am hoping that you know what could be causing this.

![image](https://user-images.githubusercontent.com/30256166/187052990-9050bd30-e342-4cbe-bd52-a5b45360fb19.png)

```
prompt: alien monolith on the moon
sampler: k-diffusion
steps: 50
scale: 7
seed: 1344113883
```

Google Colab (same output on dreamstudio beta)
![image](https://user-images.githubusercontent.com/30256166/187052768-c8633691-2ac0-4542-b869-8c561492f0e6.png)

Jupyter Lab
![image](https://user-images.githubusercontent.com/30256166/187052784-d280db41-21a4-44c9-afbf-2ec86589deeb.png)

**Comment by FuouM** [comment]
It is a known problem addressed by hlky
https://github.com/hlky/stable-diffusion-webui
```
def create_random_tensors(shape, seeds):
    xs = []
    for seed in seeds:
        torch.manual_seed(seed)

        # randn results depend on device; gpu and cpu get different results for same seed;
        # the way I see it, it's better to do this on CPU, so that everyone gets same result;
        # but the original script had it like this so i do not dare change it for now because
        # it will break everyone's seeds.
        xs.append(torch.randn(shape, device=device))
    x = torch.stack(xs)
    return x
```
